.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -12,22 +12,71 @@
 name: AGW Test LTE Integration With Make Containerized Build
 
 on:
-  workflow_dispatch: null
-  workflow_run:
-    workflows:
-      - AGW Build & Publish Container
-    types:
-      - completed
+  workflow_dispatch:
+    inputs:
+      digest_c:
+        type: string
+        required: true
+      digest_python:
+        type: string
+        required: true
+      digest_go:
+        type: string
+        required: true
+      registry:
+        type: string
+        required: true
+      test_targets:
+        type: choice
+        options: [ precommit, extended_tests ]
+        required: true
+  workflow_call:
+    inputs:
+      digest_c:
+        type: string
+        required: true
+      digest_python:
+        type: string
+        required: true
+      digest_go:
+        type: string
+        required: true
+      registry:
+        type: string
+        required: true
+      test_targets:
+        type: string  # It seems we cannot define a choice with options in workflow_call
+        required: true
 
 jobs:
   lte-integ-test-containerized:
-    if: github.event.workflow_run.conclusion == 'success'
     runs-on: macos-12
-    strategy:
-      fail-fast: false
-      matrix:
-        test_targets: [ "precommit", "extended_tests" ]
     steps:
+      - name: Show inputs
+        run: |
+          echo Docker image digest C      ${{ inputs.digest_c }}
+          echo Docker image digest Python ${{ inputs.digest_python }}
+          echo Docker image digest Go     ${{ inputs.digest_go }}
+          echo Docker registry            ${{ inputs.registry }}
+          echo test targets               ${{ inputs.test_targets }}
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
+      - id: set-registry
+        name: Set registry and image_prefix
+        run: |
+          echo ${{ inputs.registry }}
+          echo registry=${{ inputs.registry }} >> $GITHUB_OUTPUT
+          if [ ${{ inputs.registry }} = docker.io ]
+          then
+            echo image_prefix=${{ secrets.JFROG_USERNAME }}/ >> $GITHUB_OUTPUT  # dockerhub image URLs have the form docker.io/<username>/image
+          fi
+      - name: Write image digest to docker-compose.yaml
```

Comment:
I moved this part up because it was wrong several times and so it would show up faster ;)